test/prob/neg_binomial/neg_binomial_test.hpp

```diff
@@ -24,11 +24,11 @@ class AgradDistributionsNegBinomial : public AgradDistributionTest {
     parameters.push_back(param);
     log_prob.push_back(-142.6147368129045105434);  // expected log_prob
 
-    param[0] = 13;
-    param[1] = 1e11;
-    param[2] = 1e10;
+    param[0] = 10;
+    param[1] = 1e6;
+    param[2] = 1e5;
     parameters.push_back(param);
-    log_prob.push_back(-2.6185576442208003);  // expected log_prob
+    log_prob.push_back(-2.0785666431081630812);  // expected log_prob
```

Comment:
I made these things easier so the test tolerances could stay high.

This is code to verify:
```
options(digits = 20)
x = 10
alpha = 1e6
beta = 1e5
n = alpha
p = beta / (beta + 1)
dnbinom(x, n, p, log = TRUE)
```

I added 1e11/1e10 tests to unit/math/mix since these high values were clearly here for a reason. This will do low precision checks at the high parameter values.